Content.Client/Arcade/BlockGameMenu.cs

```diff
@@ -519,34 +520,40 @@ public void SetGameoverInfo(int amount, int? localPlacement, int? globalPlacemen
         {
             var globalPlacementText = globalPlacement == null ? "-" : $"#{globalPlacement}";
             var localPlacementText = localPlacement == null ? "-" : $"#{localPlacement}";
-            _finalScoreLabel.Text = Loc.GetString("Global") + $": {globalPlacementText}\n" + Loc.GetString("Local") + $": {localPlacementText}\n" + Loc.GetString("Points") + $": {amount}";
+            _finalScoreLabel.Text =
+                Loc.GetString("Global: {0}\nLocal: {1}\nPoints: {2}", globalPlacementText, localPlacementText, amount);
         }
 
         public void UpdatePoints(int points)
         {
-            _pointsLabel.Text = Loc.GetString("Points") + $": {points}";
+            _pointsLabel.Text = Loc.GetString("Points: {0}", points);
         }
 
         public void UpdateLevel(int level)
         {
-            _levelLabel.Text = Loc.GetString("Level") + $" {level + 1}";
+            _levelLabel.Text = Loc.GetString("Level {0}", level + 1);
         }
 
         public void UpdateHighscores(List<BlockGameMessages.HighScoreEntry> localHighscores,
             List<BlockGameMessages.HighScoreEntry> globalHighscores)
         {
-            var localHighscoreText = Loc.GetString("Station") + ":\n";
-            var globalHighscoreText = Loc.GetString("Nanotrasen") + ":\n";
+            var localHighscoreText = new StringBuilder(Loc.GetString("Station\n"));
+            var globalHighscoreText = new StringBuilder(Loc.GetString("Nanotrasen:\n"));
             for (int i = 0; i < 5; i++)
             {
-                localHighscoreText += $"#{i + 1} " + (localHighscores.Count > i
-                    ? $"{localHighscores[i].Name} - {localHighscores[i].Score}\n" : "??? - 0\n");
-                globalHighscoreText += $"#{i + 1} " + (globalHighscores.Count > i
-                    ? $"{globalHighscores[i].Name} - {globalHighscores[i].Score}\n" : "??? - 0\n");
+                if (localHighscores.Count > i)
+                    localHighscoreText.AppendLine(Loc.GetString("#{0}: {1} - {2}", i + 1, localHighscores[i].Name, localHighscores[i].Score));
+                else
+                    localHighscoreText.AppendLine(Loc.GetString("#{0}: ??? - 0", i + 1));
+
+                if (globalHighscores.Count > i)
+                    globalHighscoreText.AppendLine(Loc.GetString("#{0}: {1} - {2}", i + 1, globalHighscores[i].Name, globalHighscores[i].Score));
+                else
+                    globalHighscoreText.AppendLine(Loc.GetString("#{0}: ??? - 0", i + 1));
```

Comment:
I don't think they really need translation? But we can leave it like this if somebody wants to translate it

Reply:
Names are probably alright without a translation but numbers would need to be.